views/firstLogin/sections/thanks.hbs

```diff
@@ -1,6 +1,38 @@
 <section data-panel="section-{{section-nr}}">
-    <h2 class="h4">Danke!</h2>
-    <p>Du kannst jetzt mit der Benutzung der {{@root.theme.short_title}} fortfahren.<br>
-        <a class="btn btn-primary mt-2" data-testid='btn_schul-cloud_erkunden' href="/">{{@root.theme.short_title}} erkunden</a>
-    </p>
+    <div class="welcomeGreeting">
+        <img src="/images/firstLogin/welcome_banner.svg">
+        <div class="row mt-3">
+            <div class="col-xs-4 col-md-3">
+                <img src="/images/firstLogin/welcome_1_fobi.svg" alt="Online-Fortbildung">
+            </div>
+            <div class="col-xs-8 col-md-9">
+                <p>
+                    Wie funktioniert eigentlich die {{@root.theme.title}}?
```

Comment:
For students: Remove that paragraph and insert a cool student motivation text here.